### PR TITLE
More API views tests

### LINKED ISSUE #232: Expand tests for api_views
As @frboffi and @catherine-martlin suggested, it would be a good idea to expand the tests in `test_api_views` to include each of the five JWST instruments.  Currently, we just have tests for various data from FGS.

### PR DESCRIPTION
**Pull request by bourque**
This PR adds several tests to `test_api_views` in order to test more instruments, proposals, and filenames.  It also refactors a few of the functions in `data_containers` in order to speed them up, as I was finding some of the views to be really slow when dealing with a large amount of data.

Closes #232

### CODE REVIEW

**Comment by bourque** [comment]
@SaOgaz Do you happen to know why this PR randomly has a Travis CI build tacked on it?  It appears to have come out of nowhere.  We don't even use Travis 🤔

**Comment by bourque** [comment]
@SaOgaz Nevermind, when I updated the branch it went away 🤷‍♂️

File: jwql/website/apps/jwql/data_containers.py [hunk 1]
```diff
@@ -421,13 +421,11 @@ def get_instrument_proposals(instrument):
     """
 
     service = "Mast.Jwst.Filtered.{}".format(instrument)
-    params = {"columns": "filename",
+    params = {"columns": "program",
```
Comment (laurenmarietta) [inline jwql/website/apps/jwql/data_containers.py]: Smart!

File: jwql/website/apps/jwql/data_containers.py [hunk 2]
```diff
@@ -461,17 +459,11 @@ def get_preview_images_by_instrument(inst):
...
-            PREVIEW_IMAGE_FILESYSTEM,
-            'jw{}'.format(proposal),
-            '{}*.jpg'.format(filename))))
+    # Get list of all preview_images
+    preview_images = glob.glob(os.path.join(PREVIEW_IMAGE_FILESYSTEM, '*', '*.jpg'))
 
-    # Only return the filenames
-    preview_images = [os.path.basename(preview_image) for preview_image in preview_images]
+    # Get subset of preview images that match the filenames
+    preview_images = [item for item in preview_images if os.path.basename(item).split('_integ')[0] in filenames]
```
Comment (laurenmarietta) [inline jwql/website/apps/jwql/data_containers.py]: Wow! So much cleaner (and must be faster, too).

Comment (laurenmarietta) [reply jwql/website/apps/jwql/data_containers.py]: (This will also help with #254!)

**Pull request merged**